hwtypes/smt_bit_vector.py

```diff
@@ -643,11 +643,13 @@ def substitute(self, *subs : tp.List[tp.Tuple["SBV", "SBV"]]):
 #    def __int__(self):
 #        return self.as_uint()
 #
-#    def as_uint(self):
-#        return self._value.bv_unsigned_value()
-#
-#    def as_sint(self):
-#        return self._value.bv_signed_value()
+    def as_uint(self):
```

Comment:
could you add a comment about avoiding circular imports?

Reply:
Added comments